pkg/virt-config/virt-config.go

```diff
@@ -49,6 +49,7 @@ const (
 	DefaultPPC64LEEmulatedMachines                  = "pseries*"
 	DefaultAARCH64EmulatedMachines                  = "virt*"
 	DefaultLessPVCSpaceToleration                   = 10
+	DefaultMinimumReservePVCBytes                   = 131072
```

Comment:
The configmap is deprecated and does not get new values. Can you add this to the `KubeVirt` CR?

Reply:
Oh, I thought I had added it to both as this ends up being an API change, but perhaps I didn't figure the whole thing out. I'll review this.

Reply:
Ah now I see that you added it to the KubeVirt CR too. You will have to run `make generate` to get all the boilerplate which that involves.

Reply:
Thanks, will do!